.github/workflows/csharp-test.yml

```diff
@@ -43,6 +43,8 @@ jobs:
       - name: Run .NET tests
         working-directory: sdks/csharp
         run: dotnet test -warnaserror
+        with:
+          dotnet-version: '8.x'
```

Comment:
I think this is actually not working - the workflow file says it's broken now: https://github.com/clockworklabs/SpacetimeDB/actions/runs/18118462194